lib/do_volume/do_create.js

```diff
@@ -0,0 +1,208 @@
+                            if (net) {
+                                ctx.networks.push(net);
+                            }
+
+                            nextNet(getNetErr);
+                        });
+                }
+            }, next);
+        },
+        function createVolume(ctx, next) {
+            var createVolumeParams = {
+                type: 'tritonnfs',
+                name: opts.name,
+                networks: ctx.networks,
+                size: ctx.size
+            };
+
+            if (opts.type) {
+                createVolumeParams.type = opts.type;
+            }
+
+            self.top.tritonapi.cloudapi.createVolume(createVolumeParams,
+                function onRes(volCreateErr, volume) {
+                    ctx.volume = volume;
+                    next(volCreateErr);
+                });
+        },
+        function maybeWait(ctx, next) {
+            var distraction;
+            var waitTimeout = opts.wait_timeout === undefined ?
+                    undefined : opts.wait_timeout * 1000;
+
+            if (!opts.wait) {
+                next();
+                return;
+            }
+
+            if (process.stderr.isTTY && opts.wait.length > 1) {
+                distraction = distractions.createDistraction(opts.wait.length);
+            }
+
+            self.top.tritonapi.cloudapi.waitForVolumeStates({
+                id: ctx.volume.id,
+                states: ['ready', 'failed'],
+                timeout: waitTimeout
+            }, function onWaitDone(waitErr, volume) {
+                if (distraction) {
+                    distraction.destroy();
+                }
+
+                ctx.volume = volume;
+                next(waitErr);
+            });
+        },
+        function outputRes(ctx, next) {
+            assert.object(ctx.volume, 'ctx.volume');
+
+            if (opts.json) {
+                console.log(JSON.stringify(ctx.volume));
+            } else {
+                console.log(JSON.stringify(ctx.volume, null, 4));
+            }
+        }
+    ]}, cb);
+}
+
+do_create.options = [
+    {
+        names: ['help', 'h'],
+        type: 'bool',
+        help: 'Show this help.'
+    },
+    {
+        group: 'Create options'
+    },
+    {
+        names: ['name', 'n'],
+        helpArg: 'NAME',
+        type: 'string',
+        help: 'Volume name. If not given, one will be generated server-side.'
+    },
+    {
+        names: ['type', 't'],
+        helpArg: 'TYPE',
+        type: 'string',
+        help: 'Volume type. Default is "tritonnfs".'
+    },
+    {
+        names: ['size', 's'],
+        type: 'string',
+        helpArg: 'SIZE',
+        help: 'The `size` input parameter must match the following regular ' +
+            'expression: /(\d+)(g|m|G|M|gb|mb|GB|MB)/ All units are in ' +
+            'ibibytes (mebibytes and gibibytes). `g`, `G`, `gb` and `GB` ' +
+            'stand for "gibibytes". `m`, `M`, `mb` and `MB` stand for ' +
+            '"mebibytes".',
```

Comment:
> Perhaps change the RFD to only accept the one spelling of a unit?

Right, I didn't mean that we can't change the code, more that it is possible that we wrote that in the RFD because we had agreed that this is what we wanted to expose. Anyway, I agree it's better to expose fewer different spellings, I'm happy to change the RFD for that.